webpack/assets/javascripts/react_app/components/common/Date/Date.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedDate } from 'react-intl';
+import { timezone } from '../../../common/i18n';
+
+// 9/3/2010
```

Comment:
left over?

Reply:
Not really. I planned to keep the date format with each component just to give people better clue about what is the output. But I agree it deserves some more explanation.

Reply:
once you add jest snapshots, we could see the output ;)